frontend/src/components/DataProductLink.jsx

```diff
@@ -1,15 +1,16 @@
-import React from "react"
-import settings from "../settings"
+import React, { useContext } from "react"
+import DataspaceConfigurationContext from "../context/dataspaceConfigurationContext"
 
 export default function DataProductLink({ definition }) {
-  const link = `${settings.DATA_DEFINITION_VIEWER_URL}/definitions/${definition}`
+  const dataSpaceConfiguration = useContext(DataspaceConfigurationContext)
+
   return (
     <span>
-      <a href={link} target="_blank">
+      <a href={dataSpaceConfiguration.definitionViewerUrl} target="_blank">
         {definition}
       </a>{" "}
       data product published under <strong>ioxio</strong> source on{" "}
-      <a href={settings.IOXIO_SANDBOX_DATASPACE_URL} target="_blank">
+      <a href={dataSpaceConfiguration.dataspaceBaseUrl} target="_blank">
         IOXIO Sandbox
       </a>{" "}
       dataspace.
```

Comment:
The name of the dataspace was also supposed to come from the settings, so we can deploy this to multiple different dataspaces, right?